src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -75,6 +76,68 @@ class Btrfs < BlkFilesystem
       storage_forward :configure_snapper
       storage_forward :configure_snapper=
 
+      # @!method metadata_raid_level
+      #
+      #   Gets the metadata RAID level
+      #
+      #   @return [BtrfsRaidLevel]
+      storage_forward :metadata_raid_level, as: "BtrfsRaidLevel"
+
+      # @!method metadata_raid_level=(level)
```

Comment:
I'm not sure if this will look right in the rendered yardoc. Take a look at how these combinations of setter+getter are documented in other places using `@!attribute`. For example in `blk_filesystem.rb`.